tests/btc/transactions.test.ts

```diff
@@ -296,61 +284,52 @@ describe('bitcoin transactions', () => {
     const unspent3Value = 250000;
     const totalUnspentValue = unspent1Value + unspent2Value + unspent3Value;
 
-    const selectedUnspentOutputs: Array<UTXO> = [
+    const utxos: Array<UTXO & { address: string; blockHeight?: number }> = [
       {
-        txid: '1f2bbb92a74d379db2502e8ae7a57917041db5dc531ef54e64ca532aa9f59d8c',
-        vout: 2,
+        address: '3BMxVoc3NVt8BHakAh28WZrpQqQKCxV28U',
+        blockHeight: 794591,
         status: {
+          block_hash: '00000000000000000000c46287836d6018f1e6b2c02e33bd60e5c0681bcfe211',
+          block_height: 794591,
+          block_time: 1686905456,
           confirmed: true,
-          block_height: 123123,
-          block_time: 1677048365,
-          block_hash: '000000000000000000072266ee093771d806cc9cb384461841f9edd40b52b67f',
         },
-        value: unspent1Value,
+        txid: '555a4d6147848f6798af06a716d641a15fa118df68f995521f0327d0ad7f56e5',
+        value: 290842,
+        vout: 1,
       },
       {
-        txid: '1f2bbb92a74d379db2502e8ae7a57917041db5dc531ef54e64ca532aa9f59d8d',
-        vout: 2,
+        address: '3BMxVoc3NVt8BHakAh28WZrpQqQKCxV28U',
+        blockHeight: 794590,
         status: {
+          block_hash: '00000000000000000001f6ad0390dbb0d586662de872fbdcff8e2b4aa8fcd848',
+          block_height: 794590,
+          block_time: 1686904928,
           confirmed: true,
-          block_height: 123123,
-          block_time: 1677048365,
-          block_hash: '000000000000000000072266ee093771d806cc9cb384461841f9edd40b52b67f',
-        },
-        value: unspent2Value,
-      },
-      {
-        txid: '1f2bbb92a74d379db2502e8ae7a57917041db5dc531ef54e64ca532aa9f59d8e',
-        vout: 2,
-        status: {
-          confirmed: true,
-          block_height: 123123,
-          block_time: 1677048365,
-          block_hash: '000000000000000000072266ee093771d806cc9cb384461841f9edd40b52b67f',
         },
-        value: unspent3Value,
+        txid: '064829354f7280f9031b289faab8a4cdc2e38865d3495f3c9a0101218249194b',
+        value: 247106,
+        vout: 1,
       },
     ];
 
-    const recipient1Amount = 200000;
+    const recipient1Amount = 6000;
     const recipient2Amount = 100000;
-    const satsToSend = recipient1Amount + recipient2Amount;
+    const satsToSend = recipient1Amount;
```

Comment:
The same with this test. We should try and fix it rather than rewrite it. e.g. recipient2Amount amount is still defined but is never used. We should keep it and try fix the test.